frontend/packages/ux-editor/src/components/config/FormComponentConfig.tsx

```diff
@@ -143,19 +173,26 @@ export const FormComponentConfig = ({
             />
           );
         }
-        if (rest[propertyKey].type === 'string') {
+        if (
+          rest[propertyKey].type === 'string' ||
+          rest[propertyKey].enum ||
+          rest[propertyKey].$ref?.endsWith(getExpressionSchemaDefinitionReference('string'))
+        ) {
```

Comment:
Same comment here regarding the possibility to use `getFilteredProperties` for string?